### PR TITLE
added functionality to cart

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
https://user-images.githubusercontent.com/71331989/161551258-da5ee2f0-73a0-4e17-a0e9-69f3912260a3.mp4

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
please add the description and preview for your pull requests.

File: src/components/cart/price-card.jsx [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+         return acc + curr.quantity * curr.price;
+         }, 0)}
+          </p>
         </div>
         <div class="justify-content-sb dis-flex">
           <p class="check-out-text">Discount</p>
-          <p class="check-out-text">-Rs 4000</p>
+          <p class="check-out-text">
+          Rs{cart.reduce((acc, curr) => {
+          return acc + curr.quantity *( curr.price-curr.discountedPrice);
```
Comment (sadathshariff) [inline src/components/cart/price-card.jsx:27]: Just a suggestion to use a single reduce method to get all the values you need

File: src/backend/db/products.js [hunk 2]
```diff
@@ -106,5 +115,6 @@ export const products = [
     discountedPrice: 8000,
     topProduct: false,
     categoryName: 'tabs',
+    quantity: 1,
```
Comment (sadathshariff) [inline src/backend/db/products.js:118]: any reason for  adding quantity in DB as you can do that on the client-side itself

**Review by sadathshariff — COMMENTED** [review]
Overall the Code Looks Good and descriptive names

File: src/components/cart/price-card.jsx (hunk 1 above)
Comment (Priyanshi-Tiwari) [reply src/components/cart/price-card.jsx:27]: sounds good...I will look into it

File: src/backend/db/products.js (hunk 2 above)
Comment (Priyanshi-Tiwari) [reply src/backend/db/products.js:118]: I just thought it will be easier to manipulate this way

**Pull request merged**